modules/migration-mtc-release-notes-1-7-06.adoc

```diff
@@ -0,0 +1,30 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-06_{context}"]
+= {mtc-full} 1.7.6 release notes
+
+[id="new-features-1-7-6_{context}"]
+== New features
+
+.Implement proposed changes for DVM support with PSA in OCP 4.12
```

Comment:
```suggestion
.Implement proposed changes for DVM support with PSA in Red Hat Openshift Container Platform 4.12
```
See approved names: https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1259317633
Use the variable {product-title}. Check your attributes files to verify.